app/Models/Zone.php

```diff
@@ -25,15 +25,42 @@
 
 class Zone extends Model
 {
+    public const STATUS_DRAFT = 0;
+    public const STATUS_ACTIVE = 1;
+    public const STATUSES = [self::STATUS_DRAFT, self::STATUS_ACTIVE];
     public const ZONE_SIZE = [
-        "728x90",
-        "160x600",
-        "900x120",
-        "320x100",
-        "336x280"
+        '728x90',
+        '300x250',
+        '336x280',
+        '300x600',
+        '320x100',
+        '468x60',
+        '234x60',
+        '125x125',
+        '120x600',
+        '160x600',
+        '180x150',
+        '120x240',
+        '200x200',
+        '300x1050',
+        '250x250',
+        '320x50',
+        '970x90',
+        '970x250',
+        '750x100',
+        '750x200',
+        '750x300',
```

Comment:
Until the same values in the same order won't be available in FRONT it does not work. So please set my old values at the beginning to compatible with the front.

Reply:
They are. I changed front as well.